ref/Arithmetic_Operators.md

```diff
@@ -318,10 +324,13 @@ The following operators generate pseudo random numbers.
 #### Uniformly distributed random real number between 0 and `‹number›`: `random(‹number›)`
 
     > random(10)
-    < 7.089078226464412
-    ~ \d\.\d{3,}
+    < 7.089
+    ~ \d\.\d+
+    > random(1000)
+    < 882.4784
+    ~ \d+\.\d+
     > sum(1..1000, random(5))
-    < 2464.8003929607607
+    < 2464.8004
     ~ 2\d\d\d\.\d+
```

Comment:
This is already a weak statistical analysis. While `random(5000)` would be uniformly distributed, the sum of uniformly distributed variables is not uniform any more, but according to [this post](http://math.stackexchange.com/questions/370228/probability-distribution-of-a-sum-of-uniform-random-variables) distributed according to the [Irwin-Hall distributuon](https://en.wikipedia.org/wiki/Irwin%E2%80%93Hall_distribution) which is normal in the limit (and 1000 is pretty close to that limit, I dare say). So essentially we expect that value to be around 2500, and checking the first digit supports that claim while still leaving quite a lot of room for random deviations from the expected outcome.